webrender/shaders.ron

```diff
@@ -0,0 +1,147 @@
+[
```

Comment:
Because this file was created by us, and contains the relevant information for each shader (name of the shader source file, name of the shader, features for each variant of the shader). We process this file in the `build_gfx.rs` in build time to generate shaders.